src/graphics/PlanningView.ts

```diff
@@ -43,31 +43,33 @@ export function initPlanningView() {
     globalThis.predictionGraphics = predictionGraphics;
     containerUI.addChild(predictionGraphics);
     if (labelText) {
+      labelText.style.fontSize = 100;
       labelText.anchor.x = 0.5;
-      labelText.anchor.y = 0;
+      labelText.anchor.y = 0.5;
+      labelText.scale.x = 0.3;
+      labelText.scale.y = 0.3;
       containerUI.addChild(labelText);
     }
     if (mouseLabelText) {
+      mouseLabelText.style.fontSize = 100;
```

Comment:
The floating text looks great! All it took was increasing the font size and shrinking the scale??

Reply:
Yeah pretty much! The label text previously had a font size of ~26 I think